tests/allocator-test.cc

```diff
@@ -271,7 +272,8 @@ namespace
 		 * failure we see below isn't because we aren't allowing the revocation
 		 * state machine to advance.
 		 */
-		heap_quarantine_empty();
+		TEST(heap_quarantine_empty() != -ECOMPARTMENTFAIL,
+		     "Compartment call to heap_quarantine_empty failed");
```

Comment:
```suggestion
		TEST_EQUAL(heap_quarantine_empty(), 0, "Compartment call to heap_quarantine_empty failed");
```
As per previous comment but in the test suite we have this handy macro.